aopy/preproc/bmi3d.py

```diff
@@ -408,6 +408,8 @@ def _prepare_bmi3d_v1(data, metadata):
         data['cursor_interp'] = cursor_data_time
         metadata['cursor_interp_samplerate'] = cursor_samplerate
 
```

Comment:
```suggestion

    # In some versions of BMI3D, hand position contained erroneous data
    # caused by `np.empty()` instead of `np.nan`. The 'clean_hand_position' 
    # replaces these bad data with `np.nan`.
```